DxDispatch/src/model/JsonParsers.cpp

```diff
@@ -1130,7 +1130,18 @@ Model::BufferDesc ParseModelBufferDesc(const std::filesystem::path& parentPath,
         throw std::invalid_argument("Field 'initialValues' is required."); 
     }
 
-    if (initialValuesField->value.IsArray())
+    if (initialValuesField->value.IsString())
+    {
+        if (initialValuesField->value != "deferred")
```

Comment:
>deferred [](http://example.com/codeflow?start=42&length=8)

will update JSON schema, readme.md, and sample if general approach is agreed upon

Reply:
A simple test JSON would be good too. You can add it to the existing set of CMake tests.